react-calculator/src/components/Operators.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import Operator from '../elements/Operator';
+import { add, sub, mul, div } from '../utils/operations';
+import { OPERATORS, ERROR_MESSAGE } from '../constants';
+
+export default class Operators extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      operators: OPERATORS,
+      operator: '',
+    };
+  }
+
+  onClickOperator = (operator) => {
+    if (operator !== '=') {
+      if (this.props.isNumberStep && this.props.recordNumber !== 0) {
+        alert(ERROR_MESSAGE.OVER_INPUT_NUMBER_COUNT);
+        return;
+      }
+      this.props.setStep(false);
+      this.props.setRecordNumber(this.props.screenNumber);
+      this.setState({ operator });
+      return;
+    }
+
+    this.props.setStep(false);
```

Comment:
```suggestion
    this.props.setStep(false);
    if (operator !== '=') {
      if (this.props.isNumberStep && this.props.recordNumber !== 0) {
        alert(ERROR_MESSAGE.OVER_INPUT_NUMBER_COUNT);
        return;
      }
      this.props.setRecordNumber(this.props.screenNumber);
      this.setState({ operator });
      return;
    }
```
setStep은 조건에 상관없으니 올려도 괜찮아보이는데 어떨까요?

Reply:
순서에 대해서는 크게 고려하지 않았는데 의미적인 면에서 setStep을 위로 올리는게 좋겠네요! 앞으로도 코드 작성할 때 조금 더 신경써야겠어요